src/adapters/APIAdapter.js

```diff
@@ -3,15 +3,20 @@ import Adapter from './Adapter';
 import api from '../data/api';
 
 export default class APIAdapter extends Adapter {
-  async getPerson(id) {
-    const person = await api.getPerson(id);
+  async getPerson(id, onUpdate) {
+    let person = await api.getPerson(id);
     
+    // Fake an update 5 seconds later
+    setInterval(() => {
```

Comment:
Something like this is what we meant. I think it's fine if certain adapters can't handle updates, but i can be an optional parameter for those that do.